cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -95,25 +103,12 @@ def self.config(env)
             proxy: 'dashboard',
             headers: LANGUAGE_HEADER,
             cookies: whitelisted_cookies
-          },
-          {
-            path: %w(
-              /
-              /learn*
-              /congrats
-              /minecraft
-              /starwars
-              /playlab
-              /athletes
-            ),
-            headers: LANGUAGE_HEADER,
-            cookies: LANGUAGE_COOKIES,
           }
         ],
-        # Remaining Pegasus paths are English-only and don't require any extra headers or cookies.
+        # Remaining Pegasus paths are cached, and vary only on language and default cookies.
         default: {
-          headers: [],
-          cookies: 'none'
+          headers: LANGUAGE_HEADER,
+          cookies: DEFAULT_COOKIES
```

Comment:
This diff changes default caching behavior for all Pegasus pages so that they will be cached separately per-language by default. Since all pages will now be cache-localized by default, we no longer need to special-case separate behaviors for localized paths (`/learn*`, `/congrats` removed above).